src/generate/templates/ping-message-all.h.in

```diff
@@ -0,0 +1,46 @@
+// C++ implementation of the Blue Robotics 'Ping' binary message protocol
+
+//~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
+// THIS IS AN AUTOGENERATED FILE
+// DO NOT EDIT
+//~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
+
+#pragma once
+
+{% for definition in global %}
+#include "ping-message-{{definition}}.h"
+{% endfor %}
+
+namespace PingDeviceTypeNamespace {
```

Comment:
If you need the namespace and QNamespace (I don't think you do), I think you should consider committing this declaration to ping-viewer.

The enum nested inside the namespace seems unnecessarily excessive to the point it complicates usage.

I think it's good to have this enum, but I prefer just the enum or namespace. The same request applies to the PingMessageId enum.